fedmsg_meta_fedora_infrastructure/buildsys.py

```diff
@@ -60,6 +60,10 @@ def subtitle(self, msg, **config):
             tmpl = self._(
                 "Package list change for {package}:  '{tag}'{inst}")
             return tmpl.format(inst=inst, **msg['msg'])
+        elif 'buildsys.rpm.sign' in msg['topic']:
+            tmpl = self._('Koji imported a gpg signature for '
```

Comment:
"Koji package signed" doesn't say koji signed the package, just that a package built by koji is signed :).
I think more people might agree with me that "imported a gpg signature" makes me think of importing a signature for a new key.